packages/cloudvision-connector/src/Wrpc.ts

```diff
@@ -611,7 +627,13 @@ export default class Wrpc {
       if (status && status.code === ACTIVE_CODE) {
         this.activeStreams.add(token);
       }
-      callback(err, result, status, token, requestContext);
+      const requestId = callback(err, result, status, token, requestContext);
+      if (requestId as unknown) {
```

Comment:
why do you need to do this unsafe type cast?

Reply:
because callback returns void and I overload it

Reply:
I don't think this get and subscribe logic belongs in `subscribe`, or in wrpc. It should be in the Connector. It seems in the very near future getAndSubscribe will be a first class Aeris command